packages/coding-standards/src/Sniffs/ConstantVisibilityRequiredSniff.php

```diff
@@ -4,9 +4,17 @@
 
 namespace Shopsys\CodingStandards\Sniffs;
 
+use function array_reverse;
 use PHP_CodeSniffer\Files\File;
 use PHP_CodeSniffer\Sniffs\Sniff;
 use PHP_CodeSniffer\Util\Tokens;
+use function preg_match;
+use const T_CLASS;
+use const T_CLOSE_CURLY_BRACKET;
+use const T_CONST;
+use const T_DOC_COMMENT_OPEN_TAG;
+use const T_DOC_COMMENT_STRING;
+use const T_SEMICOLON;
```

Comment:
I really like the import of functions and constants, but I prefer it grouped by type.
Like 
import classes
import functions
import constants

Reply:
Yes I aggree. It is fixed in later commit.